### ISSUE TITLE
3rd controller as hmd tracking is still visible.

### ISSUE THREAD

**Issue by MyNameKuba** [HipsterSloth/PSMoveSteamVRBridge#66]
I put the controller mac in filtering but I can still see it above my head :(

_Wysłano z mojego Mi A2 za pomocą [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by HipsterSloth** [comment]
Ah I think I see the problem. In the plugin I convert the ":" in the controller serial number to "_" but I forgot to make the same conversion for the filter in PSMoveSteamVRBridgeConfig.json. 

Example:
"DC:0C:2D:64:1A:0E" -> "DC_0C_2D_64_1A_0E"

If you change the mac address in PSMoveSteamVRBridgeConfig.json to use "_" instead of ":" it should work. I'll fix this issue in the next release so that it won't matter.

**Comment by MyNameKuba** [comment]
So I just need to change ":" to "_" in json?

**Comment by HipsterSloth** [comment]
yeah so for example if your PSMoveSteamVRBridge.json file looks like this:

> {
	"is_valid":                              true,
	"version":                               1,
	"filter_virtual_hmd_serial":             *"DC:0C:2D:64:1A:0E"*,
	"server_address":                        "localhost",
	"server_port":                           "9512",
	"auto_launch_psmove_service":            true,
	"use_installation_path":                 true,
	"has_calibrated_world_from_driver_pose": true,
	"world_from_driver_pose.orientation.w":  1.0,
	"world_from_driver_pose.orientation.x":  0.0,
	"world_from_driver_pose.orientation.y":  0.0,
	"world_from_driver_pose.orientation.z":  0.0,
	"world_from_driver_pose.position.x":     -0.43832564,
	"world_from_driver_pose.position.y":     -1.0446831,
	"world_from_driver_pose.position.z":     -0.23072098
}`

Change it to this:

> {
	"is_valid":                              true,
	"version":                               1,
	"filter_virtual_hmd_serial":             *"DC_0C_2D_64_1A_0E"*,
	"server_address":                        "localhost",
	"server_port":                           "9512",
	"auto_launch_psmove_service":            true,
	"use_installation_path":                 true,
	"has_calibrated_world_from_driver_pose": true,
	"world_from_driver_pose.orientation.w":  1.0,
	"world_from_driver_pose.orientation.x":  0.0,
	"world_from_driver_pose.orientation.y":  0.0,
	"world_from_driver_pose.orientation.z":  0.0,
	"world_from_driver_pose.position.x":     -0.43832564,
	"world_from_driver_pose.position.y":     -1.0446831,
	"world_from_driver_pose.position.z":     -0.23072098
}`

**Comment by MyNameKuba** [comment]
Thanks :D

**Comment by MyNameKuba** [comment]
Still not working and opening again :/
My config:
{
	"is_valid":                              true,
	"version":                               1,
	"filter_virtual_hmd_serial":             "00_07_04_21_00_9d",
	"server_address":                        "localhost",
	"server_port":                           "9512",
	"auto_launch_psmove_service":            true,
	"use_installation_path":                 true,
	"has_calibrated_world_from_driver_pose": true,
	"world_from_driver_pose.orientation.w":  0.36698419,
	"world_from_driver_pose.orientation.x":  0.0,
	"world_from_driver_pose.orientation.y":  0.93022716,
	"world_from_driver_pose.orientation.z":  0.0,
	"world_from_driver_pose.position.x":     -0.33642825,
	"world_from_driver_pose.position.y":     0.0597198,
	"world_from_driver_pose.position.z":     -0.94800305
}

**Comment by el-bohno** [comment]
Any news? Appreciate your work.
